### PR TITLE
Create a method to put images into cloud storage

### PR DESCRIPTION
**Pull request by liuhantao9**

### CODE REVIEW

File: image-processing/main.py
Comment (quanganhhoang) [inline image-processing/main.py]: typo: processed

**Review by quanganhhoang — CHANGES_REQUESTED** [review]
Looks good to me! Please fix linting issues and typos before you merge.

File: image-processing/main.py
Comment (liuhantao9) [reply image-processing/main.py]: fixed

**Review by quanganhhoang — APPROVED** [review]
LGTM! FYI: If you add SIE-XX in front of your PR title, it will get linked to your JIRA task. Also please have descriptive commit message :)

**Pull request merged**
